<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>744</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_12">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_11" columnstretch="60,0">
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6" rowstretch="10,0,0">
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Raio da esfera</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="1">
                <widget class="QSlider" name="horizontalSliderRD">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLCDNumber" name="lcdNumber_4"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Rd</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Raio da elipse</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="2">
                <widget class="QLCDNumber" name="lcdNumber_5"/>
               </item>
               <item row="1" column="1">
                <widget class="QSlider" name="horizontalSliderRY">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLCDNumber" name="lcdNumber_6"/>
               </item>
               <item row="2" column="1">
                <widget class="QSlider" name="horizontalSliderRZ">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSlider" name="horizontalSliderRX">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>RdY</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLCDNumber" name="lcdNumber_7"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>RdZ</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>RdX</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Tamanho da caixa</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Z</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSlider" name="horizontalSliderX">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLCDNumber" name="lcdNumber"/>
               </item>
               <item row="2" column="2">
                <widget class="QLCDNumber" name="lcdNumber_3"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSlider" name="horizontalSliderZ">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLCDNumber" name="lcdNumber_2"/>
               </item>
               <item row="1" column="1">
                <widget class="QSlider" name="horizontalSliderY">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_7" columnstretch="30,0">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item row="0" column="1">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Cor</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="2">
                <widget class="QLCDNumber" name="lcdNumber_9"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>G</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSlider" name="horizontalSliderRed">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>B</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>R</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLCDNumber" name="lcdNumber_12"/>
               </item>
               <item row="3" column="1">
                <widget class="QSlider" name="horizontalSliderAlpha">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLCDNumber" name="lcdNumber_10"/>
               </item>
               <item row="2" column="1">
                <widget class="QSlider" name="horizontalSliderBlue">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>A</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSlider" name="horizontalSliderGreen">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLCDNumber" name="lcdNumber_11"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="ColorWindow" name="widgetColor" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="Plotter" name="widgetPlotter" native="true">
        <property name="minimumSize">
         <size>
          <width>471</width>
          <height>431</height>
         </size>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>50</y>
           <width>66</width>
           <height>431</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_10" rowstretch="0,0,0">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Slice</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSlider" name="horizontalSliderSlice">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLCDNumber" name="lcdNumber_8"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_9" columnstretch="0,0,0">
      <item row="0" column="2">
       <widget class="QLabel" name="labelSize">
        <property name="text">
         <string>50 x 50 x 50</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <widget class="QLCDNumber" name="lcdNumberLinha"/>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="lcdNumberColuna"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelDim">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPutVoxel"/>
   <addaction name="actionCutVoxel"/>
   <addaction name="actionPutBox"/>
   <addaction name="actionCutBox"/>
   <addaction name="actionPutSphere"/>
   <addaction name="actionCutSphere"/>
   <addaction name="actionPutEllipsoid"/>
   <addaction name="actionCutEllipsoid"/>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionNewBox"/>
  </widget>
  <action name="actionPutVoxel">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index2.jpeg</normaloff>:/new/prefix1/images/index2.jpeg</iconset>
   </property>
   <property name="text">
    <string>Quadro</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCutVoxel">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index1.jpeg</normaloff>:/new/prefix1/images/index1.jpeg</iconset>
   </property>
   <property name="text">
    <string>Corte quadro</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPutBox">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index5.jpeg</normaloff>:/new/prefix1/images/index5.jpeg</iconset>
   </property>
   <property name="text">
    <string>Cubo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCutBox">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index6.jpeg</normaloff>:/new/prefix1/images/index6.jpeg</iconset>
   </property>
   <property name="text">
    <string>Corte cubo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPutSphere">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index4.jpeg</normaloff>:/new/prefix1/images/index4.jpeg</iconset>
   </property>
   <property name="text">
    <string>Esfera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCutSphere">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index3.jpeg</normaloff>:/new/prefix1/images/index3.jpeg</iconset>
   </property>
   <property name="text">
    <string>Corte esfera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPutEllipsoid">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index14.jpeg</normaloff>:/new/prefix1/images/index14.jpeg</iconset>
   </property>
   <property name="text">
    <string>Elipse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCutEllipsoid">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index13.jpeg</normaloff>:/new/prefix1/images/index13.jpeg</iconset>
   </property>
   <property name="text">
    <string>Corte Elipse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRotClk">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index7.jpeg</normaloff>:/new/prefix1/images/index7.jpeg</iconset>
   </property>
   <property name="text">
    <string>RotClk</string>
   </property>
  </action>
  <action name="actionRotCClk">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index8.jpeg</normaloff>:/new/prefix1/images/index8.jpeg</iconset>
   </property>
   <property name="text">
    <string>RotCClk</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="text">
    <string>Invert</string>
   </property>
  </action>
  <action name="actionNewBox">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index12.jpeg</normaloff>:/new/prefix1/images/index12.jpeg</iconset>
   </property>
   <property name="text">
    <string>Nova tela</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="imagens.qrc">
     <normaloff>:/new/prefix1/images/index11.jpeg</normaloff>:/new/prefix1/images/index11.jpeg</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionXY">
   <property name="text">
    <string>XY</string>
   </property>
  </action>
  <action name="actionZX">
   <property name="text">
    <string>ZX</string>
   </property>
  </action>
  <action name="actionYZ">
   <property name="text">
    <string>YZ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <signal>moveX(int)</signal>
    <signal>moveY(int)</signal>
    <signal>planeChosen(int)</signal>
    <slot>mudaFrequencia(int)</slot>
    <slot>changeRed(int)</slot>
    <slot>changeGreen(int)</slot>
    <slot>changeBlue(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ColorWindow</class>
   <extends>QWidget</extends>
   <header>colorwindow.h</header>
   <container>1</container>
   <slots>
    <slot>changeRed()</slot>
    <slot>changeGreen()</slot>
    <slot>changeBlue()</slot>
    <slot>changeAlpha()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRD</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_9</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_10</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_11</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_12</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>659</y>
    </hint>
    <hint type="destinationlabel">
     <x>1078</x>
     <y>663</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>changeRed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>645</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>changeGreen(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>664</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>changeBlue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1008</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>677</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSlice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
